bin/hackney/import_categories

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env perl
```

Comment:
I see a couple of small differences, but is it worth factoring this together with the main bin/import_categories script, perhaps by adding a couple of hooks and the like? For the next time?